main.tf

```diff
@@ -1,12 +1,4 @@
 /**
- * Enable AWS GuardDuty and configures any findings to be sent to and SNS topic.
```

Comment:
You can clear out the entire set of comments here and just write the docs directly in the README

Reply:
Ask and you shall receive.